airflow/cli/commands/connection_command.py

```diff
@@ -314,7 +314,7 @@ def connections_delete(args):
 def connections_import(args):
     """Import connections from a file."""
     if os.path.exists(args.file):
-        _import_helper(args.file, args.overwrite)
+        _import_helper(args.file, args.replace)
```

Comment:
how is this change related to the PR?

Reply:
That's due to the refactoring I did, seems like Pycharm went a bit to far, didn't even catch that one, thanks for pointing it out

Reply:
I've reverted back that change